schemareader/tableFilters.go

```diff
@@ -54,6 +54,18 @@ func applyTableFilters(table Table) Table {
 		virtualIndexColumns := []string{"name", "version"}
 		table.UniqueIndexes[VirtualIndexName] = UniqueIndex{Name: VirtualIndexName, Columns: virtualIndexColumns}
 		table.MainUniqueIndexName = VirtualIndexName
+	case "rhnconfigfiletype":
+		virtualIndexColumns := []string{"label"}
```

Comment:
I think this is a catalog table, so we should be able to create a unique index on the label and avoid this hack.

Reply:
@rjmateus this come as a separate issue in uyuni I guess.

Reply:
I would need your help on this one